### DISCUSSION TITLE
Is Atom vulnerable to webp bug?

### DISCUSSION THREAD

**Discussion by sjurgis** [atom-ide-community/atom#500] [Q&A]
There's a [severe webp vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2023-4863) circulating that impacts Electron apps. 

Given Atom was built on electron and haven't been updated for over a year - is it latest Atom vulnerable?

**Comment by icecream17** [comment]
potentially
A vulnerability more likely to apply is https://github.com/atom-community/atom/blob/master/package.json#L122C30-L122C70 > https://github.com/atom/markdown-preview/blob/master/package.json#L13 > https://github.com/cure53/DOMPurify/releases/tag/2.1.0 given that historically similar proof-of-concepts exist (https://statuscode.ch/2017/11/21/from-markdown-to-rce-in-atom/)

EDIT: the latter does not apply since packages don't work. But [it does apply to pulsar](https://github.com/pulsar-edit/pulsar/blob/master/packages/markdown-preview/package.json#L13).
